### DISCUSSION TITLE
The newest deepxde 1.9.0 version does not contain some commits

### DISCUSSION THREAD

**Discussion by lijialin03** [lululxvi/deepxde#1327] [General]
The latest deepxde 1.9.0 version does not contain some commits。
For example, the 451 line of deepxde/model.py is "losses = paddle.concat(losses, axis=0)" in deepxde1.9.0, which has been changed to "losses = paddle.stack(losses, axis=0)" in github commit https://github.com/lululxvi/deepxde/commit/21ca6afa855ca59dfed381a14574f3350a548125.
Examples will not run without the change above with Paddle develop after 5.10 version. Now that Paddle develop has been updated, installation package might should be updated too.
Thank you.

**Comment by lululxvi** [comment]
So, we should publish a new version?

**Reply by lijialin03** [reply]
Yes, I mean is there any possibility to include recent commits to tag1.9.0? If not, only publishing new version can solve the problem of failure to run in paddle develop.
We don't want to disrupt the release plan of deepxde itself, so it would be nice if there is a better way to solve the problem.

**Reply by lululxvi** [reply]
A new version will be released.
